src/partials/pages/_3d-modeling.html

```diff
@@ -32,8 +24,8 @@ <h1 class="modeling-hero__title">
 </section>
 
 <!-- 3D MODELING CREATIONS -->
-<section class="section section-creations">
-  <div class="container bgi-mobile bgi">
+<section class="section">
+  <div class="container bgi">
```

Comment:
bgi is not descriptive and is not repeated further in classes naming - change it to methods

logically section should contain methods class
+container must contain methods__container mix

Reply:
Maryna? please fix this one.

Reply:
it was fixed in second commit

Reply:
Please leave comments when doing changes - it's hard for me to see them other vice.